apps/deeplearning_apps/style_transfer/README.md

```diff
@@ -0,0 +1,44 @@
+<h1 align="center" id="title">Style Transfer Wave App</h1>
```

Comment:
Rewrite this using template README. Remove all the demo videos and images, just use one gif.